components/setup/BuildInsruction.tsx

```diff
@@ -56,9 +69,38 @@ const BuildInstruction: React.FC<BuildInstructionProps> = ({ selectedHosting, us
     </div>
     }
 
+    const AdditionalInstructions: React.FC<AdditionalInstructionsProps> = ({ selectedInstallationType, selectedProvider }) => {
```

Comment:
`AdditionalInstructions` is not a good name. It doesn't give me any information about the purpose of the instructions or when/where should I use this component.